Backpack/Button/Classes/BPKButton.h

```diff
@@ -69,6 +69,8 @@ NS_SWIFT_NAME(Button) IB_DESIGNABLE @interface BPKButton : UIButton
 @property(nullable, nonatomic, strong) UIColor *secondaryBackgroundColor UI_APPEARANCE_SELECTOR;
 @property(nullable, nonatomic, strong) UIColor *secondaryBorderColor UI_APPEARANCE_SELECTOR;
 @property(nullable, nonatomic, strong) UIColor *linkContentColor UI_APPEARANCE_SELECTOR;
+@property(nullable, nonatomic, strong) NSNumber *defaultCornerRadius UI_APPEARANCE_SELECTOR;
```

Comment:
Dunno if it's consistent, but since the value isn't a percentage or ratio I think it's sensible to have different values. Some radii are perfect for the default size, but too small for the large buttons. One for @willhg to weigh in on though

Reply:
Discussed this with Will and agreed to trim it back down to a single property